src/lib/utils/gh.ts

```diff
@@ -44,3 +44,35 @@ export const getLastModifiedDateByPath = (path: string): string => {
   const logInfo = getGitLogFromPath(path)
   return extractDateFromGitLogInfo(logInfo)
 }
+
+const LABELS_TO_SEARCH = ["content", "design", "dev", "doc", "translation"]
+const LABELS_TO_TEXT = {
+  content: "content",
+  design: "design",
+  dev: "dev",
+  doc: "docs",
+  translation: "translation",
+}
+
+// Given a list of labels, it returns a string with the labels that match the
+// LABELS_TO_SEARCH list, using the LABELS_TO_TEXT values
+// Example:
+// - ["content :pencil:", "ux design"] => "content, design"
+// - ["documentation :emoji:", "dev required", "good first issue"] => "docs, dev"
+export const rawLabelsToText = (labels: string[]) => {
+  return labels
+    .map((label) => {
+      const labelIndex = LABELS_TO_SEARCH.findIndex((l) =>
+        label.toLocaleLowerCase().includes(l)
+      )
+
+      if (labelIndex === -1) {
+        return
+      }
+
+      const labelMatched = LABELS_TO_SEARCH[labelIndex]
+      return LABELS_TO_TEXT[labelMatched]
+    })
+    .filter(Boolean)
+    .join(", ")
+}
```

Comment:
```suggestion
export const rawLabelsToText = (labels: string[]) => {
  const list = labels
    .map((label) => {
      const labelIndex = LABELS_TO_SEARCH.findIndex((l) =>
        label.toLocaleLowerCase().includes(l)
      )

      if (labelIndex === -1) {
        return
      }

      const labelMatched = LABELS_TO_SEARCH[labelIndex]
      return LABELS_TO_TEXT[labelMatched]
    })
    .filter(Boolean)

  return new Intl.ListFormat(
    DEFAULT_LOCALE,
    { style: "long", type: "conjunction" }
  ).format(list),
}
```

Just a small suggestion if we want to make the list a little more "readable" with natural language. (An array of ["dev", "content", "design"] would turn into "dev, content and design"... [Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat))

Reply:
Won't implement this for now since we are going to just show one label in the new implementation. Great idea though, didn't know about it, thanks for sharing.